wordpress-mobile/WordPress-iOS
# Remove IPHONEOS_DEPLOYMENT_TARGET for all Pods by letting Pod targets inherit deployment target from main app

Fixes #15164. This PR takes the same approach as we've used for WooCommerce iOS: https://github.com/woocommerce/woocommerce-ios/pull/3062/files. As such, I'm copying the relevant parts of that PR description to here:

Xcode 12 issues warnings due to the deployment target of the Pods being different from the deployment target declared in the main binary. This PR adds a post_install phase to the Podfile to remove the `IPHONEOS_DEPLOYMENT_TARGET` setting from all the Pod's target configurations.

### Testing

- Checkout the branch
- Run `rake dependencies`
- Build the project. There should not be any `IPHONEOS_DEPLOYMENT_TARGET` warnings present in Xcode.

PR submission checklist:
